xled/control.py

```diff
@@ -410,6 +416,36 @@ def set_timer(self, time_on, time_off, time_now=None):
         required_keys = [u"code"]
         assert all(key in app_response.keys() for key in required_keys)
 
+    def send_realtime_frame(self, leds_number, bytes_per_led, data):
```

Comment:
The performance of this is surprisingly good, but it could be improved even more by not opening a new socket connection each time but having a long-lasting socket connection served by a daemon thread and a Queue.

I don't want to delay this pull request but I could knock this out pretty fast once this is committed.